nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/servlets/ContentAcknowledgmentServlet.java

```diff
@@ -60,20 +62,28 @@ public void init(final ServletConfig config) throws ServletException {
     }
 
     @Override
-    protected void doDelete(final HttpServletRequest request, final HttpServletResponse response) throws ServletException, IOException {
-        final X509Certificate[] certs = (X509Certificate[]) request.getAttribute("javax.servlet.request.X509Certificate");
-        String foundSubject = DEFAULT_FOUND_SUBJECT;
-        if (certs != null && certs.length > 0) {
-            for (final X509Certificate cert : certs) {
-                foundSubject = cert.getSubjectDN().getName();
-                if (authorizedPattern.matcher(foundSubject).matches()) {
-                    break;
-                } else {
-                    logger.warn(processor + " rejecting transfer attempt from " + foundSubject + " because the DN is not authorized");
-                    response.sendError(HttpServletResponse.SC_FORBIDDEN, "not allowed based on dn");
-                    return;
-                }
+    protected void doGet(final HttpServletRequest request, final HttpServletResponse response) throws ServletException, IOException {
+        if (request.getRequestURI().endsWith("/holds/ids")) {
```

Comment:
Since ListenHTTP supports a configurable base path, recommend making this path a configurable property.